Sources/Calendar.php

```diff
@@ -293,6 +293,8 @@ function CalendarPost()
 		'%T' => '%l:%M',
 	));
 
+	$time_string = preg_replace('~:(?=\s|$)~', '', $time_string);
```

Comment:
Only checking for a space or end of string could still produce `9:45:am`, `9:45:AM`, `9:45:EST`  or `9:45:-5000` in the output if the original time format contained something like `:%S%p`, `:%S%P`, `:%S%z`, or `:%S%Z` (no space after `%S`)

The following should fix that.
```suggestion
	$time_string = preg_replace('~:(?=\s|$|%[pPzZ])~', '', $time_string);
```

Reply:
Thanks